### PR TITLE
Workflow Automated Test - LRQA-67508 | master

### PR DESCRIPTION
**Pull request by evanilson-santana**

### CODE REVIEW

**Comment by pedro-michel** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/macros/WorkflowMetricsProcessListView.macro [hunk 1]
```diff
@@ -10,4 +10,17 @@ definition {
...
+		var key_workflowSearchName = "${workflowSearchName}";
+
+		Type(
+			key_fieldLabel = "Search for",
+			locator1 = "WorkflowMetricsProcessListView#METRICS_WORKFLOW_SEARCH_BAR",
+			value1 = "${workflowSearchName}");
+
+		Click(locator1 = "WorkflowMetricsProcessListView#METRICS_WORKFLOW_SEARCH_BAR_BUTTON");
+
+		Click(locator1 = "WorkflowMetrics#FILTER_CLEAR_ALL");
```
Comment (CalumR23) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/macros/WorkflowMetricsProcessListView.macro:23]: Can you add an assert to assert that the clear was successful please?

Comment (evanilson-santana) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/macros/WorkflowMetricsProcessListView.macro:39]: Hi @CalumR23 I did a verification where it says that the "Clear All" element is not present, this logic states that the clean was successful

Comment (CalumR23) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/macros/WorkflowMetricsProcessListView.macro:39]: Awesome thank you!

**Comment by CalumR23** [comment]
Forwared to BChan [Pull Request](https://github.com/CalumR23/liferay-portal/pull/6)

**Pull request closed without merging**